### ISSUE TITLE
Bug: Pet route. 

### ISSUE THREAD

**Issue by GracieEveritt** [bcantello/GA-Project-3-Backend#39]
Missing user_id, age, breed and comments. Probably just manually add for now. https://fetch-app-heroku.herokuapp.com/pets/5e9e7eb3a50d1c00041c58b1

**Comment by carlynicholson** [comment]
I'm seeing it as showing all but comments. Not sure comments section shows if it's empty? 
<img width="616" alt="Screen Shot 2020-04-21 at 9 06 22 AM" src="https://user-images.githubusercontent.com/7725531/79875569-7396ce80-83af-11ea-8e7c-ddcadf435f91.png">

**Comment by carlynicholson** [comment]
Was trying to troubleshoot the comments idea, thinking I could fill it out in the seed file and then see if it populates. I created a new local branch, spun up http://localhost:5150/, and went to http://localhost:5150/pets/5e9e7eb3a50d1c00041c58b1. I get the following:
<img width="555" alt="Screen Shot 2020-04-21 at 9 41 03 AM" src="https://user-images.githubusercontent.com/7725531/79879569-7c3dd380-83b4-11ea-8789-67e415dcc82d.png">

Tried reseeding my DB and it still won't work. Don't seem to be having issues with users or providers or even appointments. @bcantello know what I might be missing?

**Comment by carlynicholson** [comment]
Resolved. Added `comments: ""` to line 22 of `dbSeed.js` file. This section will now show up whether the comments option has been filled out or not. 

For the purpose of our demo, I went ahead and filled out a comment section for our demo dog. 

---

Without filled `comments` section: 

<img width="498" alt="Screen Shot 2020-04-21 at 10 07 37 AM" src="https://user-images.githubusercontent.com/7725531/79882601-7d70ff80-83b8-11ea-8ddd-b2da9e7d6b1c.png">

*With* filled `comments` section: 

<img width="767" alt="Screen Shot 2020-04-21 at 10 08 34 AM" src="https://user-images.githubusercontent.com/7725531/79882635-895cc180-83b8-11ea-8092-c0a862bad36e.png">
